web/src/javascript/game.js

```diff
@@ -68,13 +68,13 @@ export function getPlayerColor(players, currentPlayer) {
   return constants.playerColors[playerPosition];
 }
 
-export function drawPlayersOnMap(players, canvas) {
+export function drawPlayersOnMap(players, canvas, names) {
```

Comment:
This could be
```javascript
export function drawPlayersOnMap(players, canvas, names = "") {
```
By doing so, you'll be able to call `drawPlayerOnMap(players, canvas)` when you don't want to show the names.